rules/windows/lateral_movement_incoming_wmi.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2020/11/15"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies processes executed via Windows Management Instrumentation (WMI) on a remote host. This could be indicative of
+adversary lateral movement but may be noisy if administrators use WMI to remotely manage hosts.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "WMI Incoming Lateral Movement"
+risk_score = 47
+rule_id = "f3475224-b179-4f78-8877-c2bd64c26b88"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan = 5s
```

Comment:
Since both of these events happen a lot, this could FP just by being unlucky with timing.
Think we could shrink `maxspan` to something smaller, like 1s or 2s?

Reply:
We could also add a 4624 incoming?
Then you'd have:
`sequence [logon] [RPC endpoint mapper] [wmi child process]`

Reply:
yes it works  also with 1s, will set it to 2s, for now 2 sequence is quite reasonable performance wise and also the 2 events provide emough info (username, srcIP and childproc). once we have EQL and ECS stability we can tune all events that need logon/logoff events. 

![image](https://user-images.githubusercontent.com/64742097/99319980-db8c9500-286a-11eb-9a9b-fc2bb23192e7.png)